crates/ide/src/rename.rs

```diff
@@ -131,6 +160,46 @@ pub(crate) fn will_rename_file(
     Some(change)
 }
 
+// FIXME: Allow usage with `extern crate`?
```

Comment:
Yes, we should support that all the same

Reply:
It seems that this is also blocked on `usage` not being able to find references to `extern crate` modules.